service/src/main/java/org/cbioportal/genome_nexus/service/internal/VariantAnnotationServiceImpl.java

```diff
@@ -200,12 +249,19 @@ private EnrichmentService initPostEnrichmentService(String isoformOverrideSource
             AnnotationEnricher enricher = new HotspotAnnotationEnricher(hotspotService, true);
             postEnrichmentService.registerEnricher("cancerHotspots", enricher);
         }
+
         if (fields != null && fields.contains("mutation_assessor"))
         {
             AnnotationEnricher enricher = new MutationAssessorAnnotationEnricher(mutationAssessorService);
             postEnrichmentService.registerEnricher("mutation_assessor", enricher);
         }
 
+        if (fields != null && fields.contains("annotation_summary"))
```

Comment:
Annotation summary (which mainly contains all the fields that `genome-nexus-annotation-pipeline` needs) as an optional field. There is also a separate endpoint where one can get just the summary.